modules/ocf_desktop/files/xsession/print-notify-listener

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+
+function clean_up {
+    jobs -pr | xargs pgrep -P | xargs sudo -u ocfbroker kill
+    exit 0
+}
+
+trap clean_up INT TERM EXIT
+
+pipe="$HOME/tmp/print-notify"
+icon=/opt/share/xsession/images/ocf-color-256.png
+
+[[ -p "$pipe" ]] && rm "$pipe"
+mkfifo $pipe
```

Comment:
Does this pass shellcheck? If not, then shouldn't we hook shellcheck into this repo...

Reply:
Yeah, whoops. Missed another :(